sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -1111,63 +1117,63 @@
+     * <p>The list operations {@link CertificateAsyncClient#listIssuers()} return the {@link PagedFlux} containing
+     * {@link IssuerProperties issuerProperties} as output excluding the properties like accountId and organization details of the certificate issuer.
+     * This operation can then be used to get the full certificate issuer with its properties from {@code issuerProperties}.</p>
      *
-     * {@codesnippet com.azure.security.keyvault.certificates.CertificateAsyncClient.getCertificateIssuer#issuerBase}
+     * {@codesnippet com.azure.security.keyvault.certificates.CertificateAsyncClient.getIssuer#issuerProperties}
      *
-     * @param issuerBase The {@link IssuerBase base issuer} holding attributes of the certificate issuer being requested.
-     * @throws ResourceNotFoundException when a certificate with {@link IssuerBase#name() name} doesn't exist in the key vault.
-     * @throws HttpRequestException if {@link IssuerBase#name() name} is empty string.
+     * @param issuerProperties The {@link IssuerProperties issuerProperties} holding attributes of the certificate issuer being requested.
+     * @throws ResourceNotFoundException when a certificate with {@link IssuerProperties#getName() name} doesn't exist in the key vault.
+     * @throws HttpRequestException if {@link IssuerProperties#getName() name} is empty string.
      * @return A {@link Mono} containing the requested {@link Issuer certificate issuer}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Issuer> getCertificateIssuer(IssuerBase issuerBase) {
-        return withContext(context -> getCertificateIssuerWithResponse(issuerBase.name(), context)).flatMap(FluxUtil::toMono);
+    public Mono<Issuer> getIssuer(IssuerProperties issuerProperties) {
+        return withContext(context -> getIssuerWithResponse(issuerProperties.getName(), context)).flatMap(FluxUtil::toMono);
     }
 
     /**
-     * Gets information about the certificate issuer which represents the {@link IssuerBase} from the key vault. This operation
+     * Gets information about the certificate issuer which represents the {@link IssuerProperties} from the key vault. This operation
      * requires the certificates/manageissuers/getissuers permission.
      *
-     * <p>The list operations {@link CertificateAsyncClient#listCertificateIssuers()} return the {@link PagedFlux} containing
-     * {@link IssuerBase base issuer} as output excluding the properties like accountId and organization details of the certificate issuer.
-     * This operation can then be used to get the full certificate issuer with its properties from {@code issuerBase}.</p>
+     * <p>The list operations {@link CertificateAsyncClient#listIssuers()} return the {@link PagedFlux} containing
+     * {@link IssuerProperties issuerProperties} as output excluding the properties like accountId and organization details of the certificate issuer.
```

Comment:
Don't do this where you lower case the class name when linking to it. Just use the default link text.